stock-tracker/src/socket/eventActions.js

```diff
@@ -0,0 +1,45 @@
+import {
```

Comment:
Pref: I would _personally_ group all this into a namespaced import:
```
import * as EventType from "./eventTypes";
```
and then use them like
```
[EventType.STOCK_DATA, setResponseAction]
```